lab-michael/gulpfile.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const mocha = require('gulp-mocha');
+
+
+
+gulp.task('welcome', function() {
+  console.log('Welcome to the Single Resource API');
+});
+
+gulp.task('test', function() {
+  gulp.src('./test/test-*.js', {read: false})
+  .pipe(mocha({reporter: 'nyan'}));
+});
+
+gulp.task('eslint', function() {
+  return gulp.src(['**/*.js', '!node_modules/**'])
+  .pipe(eslint({ fix: true}))
```

Comment:
I mentioned this a while back, but you should probably be wary of automatically fixing linter issues unless you know exactly what your linter is testing for and what those fixes are going to do to your code base.